src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -220,6 +220,7 @@
   <div class="item" id="version-list"
        data-stats="{{ url('devhub.versions.stats', addon.slug) }}">
     <div class="item_wrapper">
+      {% if addon.status != amo.STATUS_DISABLED %}{# i.e. Admin disabled #}
```

Comment:
Given that we are having comments like those a few times now I wonder if the status should be called `STATUS_ADMIN_DISABLED` tbh but that's probably part of another discussion or simply just me thinking out loud.